docker-compose.dev.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+
+  db:
+    image: postgres:9.4
+    volumes:
+      - ./config/application_user.sql:/docker-entrypoint-initdb.d/application_user.sql
+    env_file:
+      - dev.env
+
+  web:
+    build:
+      context: ./
+      dockerfile: Dockerfile.dev
+    working_dir: /srv/profiles
+    volumes:
+      - .:/srv/profiles
+    command: "python manage.py runserver --host=0.0.0.0"
+    ports:
+      - "5000:5000"
+    links:
```

Comment:
[`links` is deprecated](https://docs.docker.com/compose/compose-file/#links) and `depends_on` should be used for the same intent here, which is to decide the order in which containers are started

Reply:
OK, I'll update this.